### PR TITLE
Rename number in excluded and staged requests

### LINKED ISSUE #8732: Excluded request api is having 'number' - everywhere it's 'id'
docs/api/api/create_excluded_requests.xml has
```
<excluded_requests>
  <request number='33' description='invalid request'/>
</excluded_requests>
```

For the sake of consistency, this should be `id=`

### PR DESCRIPTION
**Pull request by eduardoj**
For the sake of consistency 'number' attribute was renamed to 'id', and 'number' element to 'request'.

Fixes #8732.

Co-authored-by: David Kang <dkang@suse.com>

### CODE REVIEW

File: docs/api/api/remove_staged_requests.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <requests>
-  <number>7</number>
-  <number>13</number>
+  <request>7</request>
```
Comment (coolo) [inline docs/api/api/remove_staged_requests.xml]: same here

File: docs/api/api/delete_excluded_requests.xml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <excluded_requests>
-  <number>33</number>
+  <request>33</request>
```
Comment (coolo) [inline docs/api/api/delete_excluded_requests.xml]: this should be `<request id='33'/>` if you want to be consistent

Comment (eduardoj) [reply docs/api/api/delete_excluded_requests.xml]: I only saw the initial version of [your comment](https://github.com/openSUSE/open-build-service/issues/8732#issuecomment-555484334). On it.

Comment (eduardoj) [reply docs/api/api/delete_excluded_requests.xml]: Done.

File: docs/api/api/remove_staged_requests.xml (hunk 1 above)
Comment (eduardoj) [reply docs/api/api/remove_staged_requests.xml]: Done.

**Pull request merged**